src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  height: 100%;
+  display: inline-block;
+  line-height: 60px;
+}
+
+nav a:hover {
+  color: var(--primary-color);
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+nav ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+nav li {
+  height: 100%;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling `nav li`, you should give the `li` elements a class and style them using that class.